docker/io-fast-login/env-dev

```diff
@@ -0,0 +1,50 @@
+# -------------------
+# function setup
+# -------------------
+NODE_ENV=dev
+FUNCTIONS_WORKER_RUNTIME=node
+WEBSITE_NODE_DEFAULT_VERSION=20.12.0
+
+AZURE_ENABLE_STRICT_SSL=false
+NODE_TLS_REJECT_UNAUTHORIZED=0
+
+# -------------------
+
+API_GATEWAY_PORT=${API_GATEWAY_PORT}
+
+###
+# Azure Storages Envs
+###
+FUNCTIONS_WORKER_RUNTIME=node
+QueueStorageConnection=${STORAGE_CONN_STRING}
+
+LOLLIPOP_GET_ASSERTION_API_KEY=foo
+LOLLIPOP_GET_ASSERTION_BASE_URL=foo
+
+###
+# Fast Login Audit log storage
+###
+FAST_LOGIN_AUDIT_CONNECTION_STRING=${STORAGE_CONN_STRING}
+
+###
+# Cosmos DB envs
+###
+COSMOS_CONNECTION_STRING=AccountEndpoint=${COSMOSDB_URI};AccountKey=${COSMOSDB_KEY};Database=${COSMOSDB_NAME}
+COSMOS_DB_NAME=${COSMOSDB_NAME}
+
+
+###
+# BackendInternal
+###
+FETCH_TIMEOUT_MS=5000
+BACKEND_INTERNAL_BASE_URL=foo
```

Comment:
let's default this to `io-backend` container name for `io-mock` interoperability
```suggestion
BACKEND_INTERNAL_BASE_URL=http://backend
```